package.json

```diff
@@ -46,6 +46,7 @@
     "crypto-browserify": "^3.12.0",
     "crypto-js": "^4.1.1",
     "date-fns": "^2.28.0",
+    "dayjs": "^1.11.0",
```

Comment:
do we really need `dayjs` when we already have `date-fns` installed?

Reply:
alright, i will switch to date fns instead of using dayjs